zuul.d/jobs.yaml

```diff
@@ -170,3 +170,14 @@
       - secret: galaxy_dev_secret
         name: ansible_galaxy_info
     nodeset: centos-8-1vcpu
+
+- job:
+    name: ansible-workshop
```

Comment:
what about workshops-tox-integration ? Or ansible-workshops-integration

I like the idea of include tox, to know we are using tox as the interface